src/protocols/ssh/ssh.c

```diff
@@ -318,11 +323,18 @@ void* ssh_client_thread(void* data) {
 
     /* While data available, write to terminal */
     int bytes_read = 0;
+    int timeout = 0;
     for (;;) {
 
         /* Track total amount of data read */
         int total_read = 0;
 
+        /* Set up return value for keepalives */
+        int alive = 0;
+
+        /* Timer for keepalives */
+        int sleep = 0;
```

Comment:
`sleep` is a bit of a misnomer here, since we aren't going to actually explicitly sleep for that period of time, but rather wait for additional data for no more than this amount of time.